enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/multiview/CmpFieldsNode.java

```diff
@@ -88,11 +88,11 @@ public SectionNode getNodeForElement(Object element) {
                 return this;
             }
         } else if (element instanceof CmpField[]) {
-            final List list1 = Arrays.asList(cmpFields.getCmpFields());
-            final List list2 = new LinkedList<>(Arrays.asList((CmpField[]) element));
+            final List<CmpField> list1 = Arrays.asList(cmpFields.getCmpFields());
+            final List<CmpField> list2 = new LinkedList<>(Arrays.<CmpField>asList((CmpField[]) element));
```

Comment:
second line could be simply
`final List<CmpField> list2 = new LinkedList<>(Arrays.asList((CmpField[]) element));`
analog to the first line (no generic type for `asList()`)

Reply:
done..